packages/transactions/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "outDir": "./dist",
+  "rootDir": "./src",
+  "strict": false
```

Comment:
should we move `strict` into the base?

Reply:
Hm. I don't really know what I'm doing but I removed the strict: false from everywhere as strict: true is in the base and strict seems good to start off with and we can relax it later (I don't know what it enforces .. )🤔 ?

Reply:
50e3ca6